sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -1890,6 +1890,10 @@ class Analyzer(
   object ResolveFunctions extends Rule[LogicalPlan] {
     val trimWarningEnabled = new AtomicBoolean(true)
     def apply(plan: LogicalPlan): LogicalPlan = plan.resolveOperatorsUp {
+      case RefreshFunction(UnresolvedFunc(multipartIdent)) =>
```

Comment:
we should match `UnresolvedFunc` only, in case we add more function-related commands later.

Reply:
and can we update the comment of this rule?